source/gui/__init__.py

```diff
@@ -924,3 +924,64 @@ def shouldConfigProfileTriggersBeSuspended():
 
 def _isDebug():
 	return config.conf["debugLog"]["gui"]
+
+class AskAllowUsageStatsDialog(wx.Dialog):
+	"""A dialog asking if the user whishes to allow NVDA usage stats to be collected by NV Access."""
+
+	def __init__(self, parent):
+		# Translators: The title of the dialog asking if usage data can be collected 
+		super(AskAllowUsageStatsDialog, self).__init__(parent, title=_("NVDA  Usage Data Collection"))
+		mainSizer = wx.BoxSizer(wx.VERTICAL)
+		sHelper = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
+
+		# Translators: A message asking the user if they want to allow usage stats gathering
+		message=_("In order to improve NVDA in the future, NV Access wishes to collect usage data from running copies of NVDA. "
+			"Data includes Operating System version, NVDA version, language, country of origin, plus certain NVDA configuration such as current synthesizer, braille display and braille table. " 
+			"No spoken or braille content will be ever sent to NV Access.  Please refer to the User Guide for a current list of all data collected.\n"
+			"Do you wish to allow NV Access to periodically collect this data in order to improve NVDA?")
+		sHelper.addItem(wx.StaticText(self, label=message))
+
+		bHelper = sHelper.addDialogDismissButtons(guiHelper.ButtonHelper(wx.HORIZONTAL))
+
+		remindMeButtonID=wx.NewId()
+		# Translators: The label of a button to remind the user later about performing some action.
+		remindMeButton = bHelper.addButton(self, remindMeButtonID, label=_("Remind me &later"))

```

Comment:
I would expect the "remind me later" button to come last, in place of "cancel". Eg "Yes, No, Cancel" -> "Yes, No, Remind me later". Its fine for it to have focus.

To make it last, make it the last call to `addButton`